Sources/Asteroids/Components/Animation.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  Animation.swift
+//  FirebladeECSDemo
+//
+//  Created by Igor Kravchenko on 18.11.2020.
+//
+
+import FirebladeECS
+
+final class Animation: ComponentInitializable {
+    var animation: Animatable
+
+    init(animation: Animatable) {
+        self.animation = animation
+    }
+
+    // swiftlint:disable unavailable_function
+    required init() {
+        fatalError("use init(animation:) instead")
```

Comment:
Please provide a default animation rather than crashing the app if used like this. For example just spinning or even doing nothing would be better than just crashing.

Reply:
I've added a spinning star animation.